llama_stack/providers/inline/vector_io/sqlite_vec/sqlite_vec.py

```diff
@@ -287,6 +286,80 @@ def _execute_query():
             scores.append(score)
         return QueryChunksResponse(chunks=chunks, scores=scores)
 
+    async def query_hybrid(
+        self,
+        embedding: NDArray,
+        query_string: str,
+        k: int,
+        score_threshold: float,
+    ) -> QueryChunksResponse:
+        """
+        Hybrid search: run keyword (FTS5) search first to get candidate IDs,
+        then run vector similarity search limited to those candidates.
+
+        Args:
+            embedding: The query embedding vector
+            query_string: The text query for keyword search
+            k: Number of results to return
+            score_threshold: Minimum similarity score threshold
+        """
+
+        def _execute_hybrid_query():
+            connection = _create_sqlite_connection(self.db_path)
+            cur = connection.cursor()
+            try:
+                emb_list = embedding.tolist() if isinstance(embedding, np.ndarray) else list(embedding)
+                emb_blob = serialize_vector(emb_list)
+
+                # Step 1: Get top candidate IDs from FTS5
+                # This can be made configurable in the future.
+                num_candidates = k * 10
```

Comment:
can we make this configurable in `sqlite_vec/config.py`?

we can default the value there imo.